resources/views/game-details.blade.php

```diff
@@ -0,0 +1,28 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Game Details</title>
+</head>
+<body>
+    <h1>Game Details</h1>
+    <table>
+        <thead>
+            <th>Details</th>
+        </thead>
+        <tbody>
+        @foreach($games as $game)
```

Comment:
in this view, you only have one `$game` object, not a list of games. So the idea here is to display detailed information about the game, rather than the same thing we show in the game-list view.